tracing-distributed/README.md

```diff
@@ -0,0 +1,18 @@
+# tracing-distributed
```

Comment:
link to the release docs, master docs, and crates.io here -- all of them have badges. See https://github.com/calibra/cargo-guppy/blob/master/guppy/README.md for an example